### ISSUE TITLE
Research financial ROI for grants (vs bounties.)

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/.github#61]
Start with figuring out GitCoin numbers. Otherwise research other grant platforms. The goal is to understand the profits of providing bounties vs grants. Please report the ratio of revenues (and profits, ideally) comparing grants vs bounties.

https://ubiquitydao.notion.site/DevPool-Financing-38a2f017acea4562959ba28e7e3558c1

**Comment by Draeieg** [comment]
https://gov.gitcoin.co/t/an-interactive-model-for-gitcoin-revenue-growth-forecasting/12625

Tl;Dr a 2-5% fee, at 50M on grants equals 1-2.5M

however, [here](https://support.gitcoin.co/gitcoin-knowledge-base/gitcoin-grants-program-1/general-questions/how-much-volume-has-gitcoin-grants-done) there is 51.7M in results vs 57.8M in "funded" which means there is a small 6.1 million discrepancy 

Previously Gitcoin stated a 72 million figure overall "funded to open source" assuming this is a mix between grants and  bounties then we have 20M worth of bounties, at the 10% rate we know they have would equal to 2M

this means profits would be roughly equal for both sector with one profiting from volume and another from net gains
